chaos_kitten/brain/orchestrator.py

```
@@ -32,666 +20,356 @@
+    toys_dir = os.path.join(os.path.dirname(os.path.dirname(__file__)), "toys")
+    bl_profiles = []
+    
+    # Try to load business logic profiles
+    bl_files = ["race_condition.yaml", "workflow_bypass.yaml", "authorization_bypass.yaml", "price_manipulation.yaml"]
+    for bl_file in bl_files:
+        profile_path = os.path.join(toys_dir, bl_file)
+        if os.path.exists(profile_path):
+            try:
+                profile = load_yaml(profile_path)
+                bl_profiles.append(profile)
+            except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)